src/Uno.UWP/ApplicationModel/Background/BackgroundTaskRegistration.cs

```diff
@@ -0,0 +1,11 @@
+﻿
```

Comment:
Then I would get:
Error	CS8618	Non-nullable property 'Trigger' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.	Uno (MonoAndroid12.0)	

But I don't want to create constructor (as it would be false implementation), and I can't change string to string? (as UWP definition is string, not string?).

Reply:
Done via internal ctor.